* grobian@gentoo.org: the use of nanosleep requires librealtime (-lrt)
                      on Solaris, OpenSolaris however doesn't need it
                      anymore

--- configure
+++ configure
@@ -1437,6 +1437,7 @@
         SHFLAGS='-shared -Wl,-h,$$(@F)'
         network_extralibs="-lsocket -lnsl"
         add_cflags -D__EXTENSIONS__
+        osextralibs="-lrt"
         ;;
     netbsd)
         oss_demuxer_extralibs="-lossaudio"
